src/Phantoman.php

```diff
@@ -15,7 +15,10 @@
 
 class Phantoman extends \Codeception\Platform\Extension
 {
-    // list events to listen to
+    /// Number of max checks while waiting for PhantomJS to be responsive/stopped
+    const MAX_CHECKS = 10;
```

Comment:
@tytoninae If you go ahead and make this a config option, the documentation for it can be added to the [basic section](https://github.com/site5/phantoman#basic) in the readme. It might make sense to have this as a config with the default to 10 if you're down to make that change.